File: PRAKTIKUM MODUL 6/PRAK605_2310817220028_SheilaSabina.py
print("Input")
s = int(input())

matrixA = []
print("Matriks A")
for i in range(s):
    row = list(map(int, input().split()))
    matrixA.append(row)

matrixB = []
print("Matriks B")
for i in range(s):
    row = list(map(int, input().split()))
    matrixB.append(row)

result = [[0 for _ in range(s)] for _ in range(s)]

for i in range(s):
    for j in range(s):
        for k in range(s):
            result[i][j] += matrixA[i][k] * matrixB[k][j]

print("\nOutput")
print("Matriks AXB")
for i in range(s):
    for j in range(s):
        print(result[i][j], end=" ")
    print()

File: PRAKTIKUM MODUL 6/PRAK605_2310817220028_SheilaSabina.c
#include <stdio.h>

int main() {
    int s;

    printf("Input\n");
    scanf("%d", &s);

    int matrixA[s][s];
    int matrixB[s][s];
    int result[s][s];

    printf("Matriks A\n");
    for (int i = 0; i < s; i++) {
        for (int j = 0; j < s; j++) {
            scanf("%d", &matrixA[i][j]);
        }
    }

    printf("Matriks B\n");
    for (int i = 0; i < s; i++) {
        for (int j = 0; j < s; j++) {
            scanf("%d", &matrixB[i][j]);
        }
    }

    for (int i = 0; i < s; i++) {
        for (int j = 0; j < s; j++) {
            result[i][j] = 0;
            for (int k = 0; k < s; k++) {
                result[i][j] += matrixA[i][k] * matrixB[k][j];
            }
        }
    }

    printf("\nOutput\n");
    printf("Matriks AXB\n");
    for (int i = 0; i < s; i++) {
        for (int j = 0; j < s; j++) {
            printf("%d ", result[i][j]);
        }
        printf("\n");
    }

    return 0;
}